pkg/asset/manifests/kubevirt/cloudproviderconfig.go

```diff
@@ -0,0 +1,34 @@
+package kubevirt
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// CloudProviderConfig is the kubevirt cloud provider config
+type CloudProviderConfig struct {
+	// The namespace in the infra cluster that the cluster resources are created in
+	Namespace string
+	InfraID   string
+}
+
+type config struct {
+	// The namespace in the infra cluster that the cluster resources are created in
+	Namespace string `json:"namespace" yaml:"namespace"`
```

Comment:
When the json and yaml fields are the same, you can omit the yaml tags.

Reply:
Removed the yaml tags